kernels/InfoKernel.cpp

```diff
@@ -222,6 +239,20 @@ void InfoKernel::makePipeline()
         stage = m_statsStage =
             &m_manager.makeFilter("filters.stats", *stage, filterOptions);
     }
+
+    if (m_stac)
+    {
+        // filters required for stac: metadata, stats,
+        if (stage != m_statsStage)
+        {
+            Options stacOps;
+            if (m_enumerate.size())
+                stacOps.add({"enumerate", m_enumerate});
+            m_stacStage = stage = &m_manager.makeFilter("filters.stats", *stage, stacOps);
```

Comment:
It is possible we already have a `filters.stats` added to the pipeline at this point if we're doing `--all` as suggested above.

Reply:
Yeah, this should account for that with the stage boolean check above. Could definitely make it more obvious by just saying `if (m_showStats)` though.